packetbeat/docs/fields.asciidoc

```diff
@@ -1115,6 +1115,16 @@ Unless this field is disabled, it always has a value, and it matches the `client
 
 
 [float]
+=== client_location
```

Comment:
I think this somehow sneaked in. Try to run `make clean` and then `make update`. I also had this once in the past but not sure what happened. Probably going back and forth from other branches and have some not checked in files which are outdated.